tests/modules/nf-core/falco/test.yml

```diff
@@ -0,0 +1,29 @@
+- name: falco test_fastqc_single_end
+  command: nextflow run ./tests/modules/nf-core/falco -entry test_fastqc_single_end -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/falco/nextflow.config
+  tags:
+    - falco
+  files:
+    - path: output/falco/fastqc_data.txt
+      md5sum: c5d0cd529d1d2f5a842bf3afd06a9567
+    - path: output/falco/fastqc_report.html
+      contains: "[ # TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead ]"
+    - path: output/falco/summary.txt
+      md5sum: 3367c81d60e4b39c89b28acd0434c8aa
+
+- name: falco test_fastqc_paired_end
+  command: nextflow run ./tests/modules/nf-core/falco -entry test_fastqc_paired_end -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/falco/nextflow.config
+  tags:
+    - falco
+  files:
+    - path: output/falco/test_1.fastq.gz_fastqc_data.txt
+      md5sum: 36d989bb9e2d5a632e19452f4e6c2a4e
+    - path: output/falco/test_1.fastq.gz_fastqc_report.html
+      contains: "[ # TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead ]"
```

Comment:
One can either remove this and keep the `md5sum` if the `md5sum` does not differ every time running the module